src/features/layout/nav/NavBar.tsx

```diff
@@ -0,0 +1,33 @@
+import { NavLink } from 'react-router-dom';
+
+const NavBar = (prop: {
+  t: any;
+  className: string;
+  handleNav?: () => void;
+}) => {
+  return (
+    <>
+      {[
+        [`${prop.t('products')}`, '/product-series'],
+        [`${prop.t('about')}`, '/about'],
+        [`${prop.t('service')}`, '/service'],
+        [`${prop.t('contact')}`, '/contact'],
+      ].map(([title, link]) => (
+        <NavLink
+          to={link}
+          key={title}
+          onClick={prop.handleNav}
+          className={({ isActive }) =>
+            isActive
+              ? `${prop.className} md:text-lg sm:text-base font-medium`
+              : prop.className
+          }
+        >
+          {title}
```

Comment:
Perhaps you could translate it here if possilbe, then the array could be just [products, about, service, contact], and the routes could be set accordingly

Reply:
Yes it looks a lot better now. Thanks!